### DISCUSSION TITLE
Layout files for AI Deck

### DISCUSSION THREAD

**Discussion by darren-chiu** [bitcraze/discussions#1470] [General]
Hello! 

I am looking to adapt a new camera for the current AI deck and was wondering if there are any kicad layout files that are publicly available? I was able to find the schematics [in the hardware repo](https://github.com/bitcraze/hardware/blob/master/src/products/ai-deck/electronics/ai-deck-revc.pdf), but was hoping to find the right pin orientations for the camera connector as their is no indicator for Pin 1 on the board. Thank you so much!

**Comment by evoggy** [comment]
Hi! Unfortunately there's no layout publicly available, just the schematics. But I've attached a screenshot of the connector from the layout here. So this is seen from the top and the top of the image is the front of the deck (i.e where the camera is). Hope this helps!

![image](https://github.com/user-attachments/assets/6c2c2dcb-a270-48fc-a5f7-2ec6bb824933)
